common/config/rush/rush-plugins.json

```diff
@@ -0,0 +1,25 @@
+/**
```

Comment:
Is there a template for this file in the `rush init` files?

Reply:
Yeah https://github.com/microsoft/rushstack/blob/master/apps/rush-lib/assets/rush-init/common/config/rush/rush-plugins.json